recipe/meta.yaml

```diff
@@ -61,7 +64,10 @@ outputs:
         - nccl         # [linux and cuda_compiler != "None"]
         - librmm       # [linux and cuda_compiler != "None"]
       run:
-        - cuda-version >={{ cuda_compiler_version }},<12  # [(cuda_compiler_version or "").startswith("11")]
+        - __cuda                   # [cuda_compiler != "None"]
```

Comment:
As the RAPIDS packages work on either CPU or GPU, we don't need this requirement

For context, the conda-forge feedstock adds this to ensure CUDA-enabled packages are only installable on a machine with a working GPU. In the RAPIDS, packages we don't need this check

```suggestion
```